frontend/src/schema.graphql

```diff
@@ -211,17 +211,19 @@ input UpdateSeriesBlock {
   order: VideoListOrder
 }
 
-type Series implements Node {
+interface Series {
```

Comment:
Interface could use some docs. Same for `WaitingSeries` and `ReadySeries` (the latter two can just refer to the interface docs tho)

Reply:
I hope you don't mind me cribbing this mostly from the comment in the migration script.